test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/DefaultTagHelperContentTest.cs

```diff
@@ -87,6 +87,65 @@ public void CanAppendContent()
         }
 
         [Fact]
+        public void CanAppendFormatContent()
```

Comment:
Might be useful to add a couple of tests to verify composite scenarios (`{0:X}`, alignment etc). This would help in the event we ever switch the underlying implementation like the bug proposes.